workbench-core/infrastructure/src/workbenchAppRegistry.ts

```diff
@@ -61,6 +65,10 @@ export class WorkbenchAppRegistry extends Construct {
       description: `Service Catalog application to track and manage all your resources for the solution ${this._solutionName}`
     });
 
+    if (this._destroy) {
```

Comment:
reason we apply removal policy twice?

Reply:
@htztomic one is for application and the other is for attributeGroup

Reply:
got it!